account_journal_lock_date/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Journal Lock Date',
+    'summary': """
+        Lock each journal independently""",
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu/',
```

Comment:
@michotm URL should be `https://github.com/OCA/account-financial-tools`

Reply:
Done ;-)